scripts/changeScanner/getChanges.js

```diff
@@ -20,7 +20,7 @@ const getChanges = () => {
 
     const isValidPackageName = validPackageNamePattern.test(fileName);
 
-    const [packageName, filePath] = isValidPackageName
+    const [packageName] = isValidPackageName
```

Comment:
```suggestion
    const packageName = isValidPackageName ? nameParts[2] : DEFAULT_PACAKGE_NAME;
```

We can replace linens 23-25 with this and remove lines 27-29